### ISSUE TITLE
lxqt.org is down: Connection reset by peer

### ISSUE THREAD

**Issue by rad164** [lxqt/lxqt.github.io#41]
at least the following sites are down:

- https://lxqt.org/
- https://blog.lxqt.org/
- https://weblate.lxqt.org/

**Comment by yan12125** [comment]
@agaida Could you have a look?

**Comment by tsujan** [comment]
His GitHub page shows he hasn't been active for a month; may be busy with life. I hope he's doing fine.

**Comment by yan12125** [comment]
Oops. Hope he is doing well.

**Comment by tsujan** [comment]
Doesn't another LXQt member have the required permissions to fix this?

**Comment by yan12125** [comment]
agaida once added my SSH key to the weblate instance, but I can no longer use the key to login :/

Meanwhile, I modified IRC channel topics on oftc.net#lxqt and oftc.ne#lxqt-dev to mention this issue.

Note to myself: `/msg ChanServ set #lxqt topic XXX`

**Comment by luis-pereira** [comment]
I Hope that everything is Ok with @agaida.

**Comment by selairi** [comment]
Hi, I don't know how is the "lxqt.org" domain controlled. Could anybody explain me?

I have done a Whois of "lxqt.org". The IP of taht server is controlled by:
[https://contabo.com](https://contabo.com)
Is this right?
We could write to that company and explain them the situation. What do you think?

Finally, "lxqt.org" is a redirect to "lxqt.github.io". We could write a note in "lxqt.github.io" explaining the situation.

**Comment by tsujan** [comment]
> ... Could anybody explain me?

I'm in the dark; relied on @agaida, who organizes things smoothly.

**Comment by mpmc** [comment]
Hi all,

I really really hope agaida and his family are okay, it's not like him to disappear without saying anything :( !

> 
> 
> Hi, I don't know how is the "lxqt.org" domain controlled. Could anybody explain me?
> 
> I have done a Whois of "lxqt.org". The IP of taht server is controlled by:
> https://contabo.com
> Is this right?
> We could write to that company and explain them the situation. What do you think?
> 
> Finally, "lxqt.org" is a redirect to "lxqt.github.io". We could write a note in "lxqt.github.io" explaining the situation.

I believe he does host everything on contabo. They may hold his contact information and I'm sure if someone were to ask nicely - **without asking for/or giving out his private info** - they may be willing to telephone him to make sure he's OK. That is the main thing.

Has anyone done this? I recommend only one person does and the could perhaps link to this issue? :)

**Comment by luis-pereira** [comment]
> Doesn't another LXQt member have the required permissions to fix this?

Really don't know.

**Comment by tsujan** [comment]
> Really don't know.

We should learn more about @agaida's work when he comes back. At least, we need to know how to make releases (→ https://github.com/lxqt/lxqt/issues/1779).

**Comment by luis-pereira** [comment]
> We should learn more about @agaida's work when he comes back. At least, we need to know how to make releases (→ [lxqt/lxqt#1779](https://github.com/lxqt/lxqt/issues/1779)).

Sure. It's a bus factor of 1.
Hope he is Ok.

**Comment by tsujan** [comment]
> Hope he is Ok.

He is, @wxl  found him yesterday, He's busy with his job.

**Comment by luis-pereira** [comment]
> He is, @wxl found him yesterday, He's busy with his job.

That's great news.

**Comment by jubalh** [comment]
> At least, we need to know how to make releases

**Comment by tsujan** [comment]
> I think we know that :)

How? If you know, please tell us at https://github.com/lxqt/lxqt/issues/1779. I know how to make releases (I do it regularly with my own projects) but not with LXQt.

**Comment by tsujan** [comment]
Of course, the development of LXQt keeps going and, most probably, the problem of releasing will be solved. But many translators — almost all of them — had got used to Weblate. The LXQt forum was also used by many.

The future of LXQt on Debian can't be bright either but I hope Ubuntu will continue packaging of next releases independently. Arch won't be affected (thanks to @yan12125).

The lesson learned: the website and Weblate aren't reliable with only one admin.

**Comment by jubalh** [comment]
> How? If you know, please tell us at lxqt/lxqt#1779. I know how to make releases (I do it regularly with my own projects) but not with LXQt.

Well the process is almost exactly the same.
But as I saw now you already answered your own question at: https://github.com/lxqt/lxqt/issues/1779#issuecomment-616203521

**Comment by tsujan** [comment]
> But as I saw now you already answered your own question at...

Yes, but I didn't sign releases of my projects before (will sign any release from now on); the importance of LXQt release signing was what I needed to know.

@agaida once [quoted](https://github.com/lxqt/libfm-qt/issues/432#issuecomment-498076457), "one should do a 'maintainer boot camp' for developers" ;)

**Comment by jubalh** [comment]
There are plenty of projects which don't GPG sign their releases. It's not a hard requirement.

**Comment by tsujan** [comment]
> If you have more questions don't hesitate to ping me.

Thanks! I'll definitely need to consult you.

**Comment by Cal8031** [comment]
lxqt.org's WHOIS changed to gandi.net on 10th April by the looks of it.

**Comment by tsujan** [comment]
A question:

@jubalh @luis-pereira @wxl 

Where should the general announcement appear (now that we don't have a site)?

I have second thoughts about the release page of `lxqt-about`. People naturally go to https://github.com/lxqt/lxqt/releases and get frightened by seeing 0.11.0 as the latest release there. What about releasing its version 0.15.0 and putting the general announcement there?

BTW, the preparation for releasing LXQt 0.15.0 is being done much faster than I thought.

**Comment by jubalh** [comment]
Good questions.

> Where should the general announcement appear (now that we don't have a site)?

Do we have any idea when we can have access to the side again?
One option could be (doesnt have to be the only one):
Put the release info in https://github.com/lxqt/lxqt.github.io/ which will result in https://lxqt.github.io/ being updated.
Then maybe we can ask agaida to at least make lxqt.org point to that github page again. Or if we think we lost lxqt.org forever we can announce https://lxqt.github.io/  as the new official website.

It has down sides and up sides. One positive point would be that we would not depend on one single person.

**Comment by tsujan** [comment]
> Put the release info in https://github.com/lxqt/lxqt.github.io/ which will result in https://lxqt.github.io/ being updated.

@jubalh, that's a very good idea but working with HTML is really beyond my time. I've written the announcement in markdown.

**Comment by jubalh** [comment]
>  I've written the announcement in markdown.

Plenty of generators out there.
For example you can use `pandoc` to create html from markdown then insert it in relevant locations.

**Comment by tsujan** [comment]
It's as follows (I'll complete it after adding pre-release PRs for the remaining components -- the most important ones are already covered):

---------------------------------------------------------------
The LXQt team announces the release of LXQt 0.15.0, the Lightweight Qt Desktop Environment.

# Highlights

## LibFM-Qt / PCManFM-Qt

 * Now, mount passwords can be saved (temporarily or permanently) if gnome-keyring is present.
 * "Single window mode" is supported.
 * Richer file tooltips.
 * Multi-screen wallpaper support .
 * Deletion time is added to Trash.
 * Support for hiding/showing thumbnails on the fly.
 * Better keyboard navigation.
 * Problems with remote folders are fixed.
 * LXQt file dialog has smarter extension handling.
 * Many fixes.

## LXQt Archiver

This is a new addition to LXQt: a fully functioning archive manager based on LibFM-Qt and integrated into PCManFM-Qt

## LXQt Panel

 * A new plugin is added to the panel for changing display backlight.
 * Now Desktop Switcher has an option to show only the active desktop.
 * Menu search is made smoother.
 * Panel positioning is corrected with multiple screens.
 * An option is added to Task Manager for moving windows to next/previous virtual desktop with mouse wheel.

## LXQt Configuration

Drag and drop of screens is improved and made easier in Monitor Settings and the Brightness Settings dialog is improved.

## QTerminal / QTermWidget

TO BE ADDED

## LXImage Qt

 * "Open With..." is added to file menu.
 * Keyboard shortcuts are made configurable.
 * The maximum number of recent files is made configurable.
 * Showing of image outline is supported.
 * Fixes.

# Release Notes

Please see the release page of each LXQt component for its detailed release note.

**Comment by tsujan** [comment]
> Plenty of generators out there.

I don't have the time. Another member should do that.

**Comment by tsujan** [comment]
Also, please note that that there are lots of release notes for individuals components, which I'll add to their release pages (they're ready). Anyone who edits that site will need to add them too (copy-paste).

**Comment by luis-pereira** [comment]
@tsujan I do agree with @jubalh.
The release notes are done in markup. Check it [here](https://github.com/lxqt/lxqt.github.io/tree/master/_posts).

**Comment by tsujan** [comment]
@luis-pereira 
I agree with with @jubalh too -- he had a good point that should be followed -- but still I don't have time to re-arrange the notes I've made. What I can do is adding them to releases before I press GitHub's release button.

After the release, an LXQt member could copy-paste release notes from GitHub release pages, change them if needed (to have a uniform style), give them appropriate names and include them in that page. I'll put the general announcement in https://github.com/lxqt/lxqt/releases. Of course, it could be reworded too if needed.

Anyway, that needs to be done *after* the whole LXQt is released -- the task I volunteered for (> 70% complete until now).

**Comment by tsujan** [comment]
Update:

All pre-release changes are made except for those of Qterminal (waiting for @yan12125's reply) and lxqt-config (waiting for @selairi's fix).

After these 2 cases are dealt with, I'll start to release components in the way explained at https://github.com/lxqt/lxqt/issues/1779. The only differences from previous releases will be that:

 1. The packages will be signed by me.
 2. Each release will have a note (I think this should have been done before -- empty notes were confusing).

I thought it would take weeks but only a few days have passed :)

**Comment by yan12125** [comment]
Finished PRs for qterm\* are just merged. Please tell me if I missed something :)

**Comment by tsujan** [comment]
@yan12125 
Thank you so much!

**Comment by tsujan** [comment]
Update:

Done: https://github.com/lxqt/lxqt/issues/1779#issuecomment-619156528

**Comment by tsujan** [comment]
FYI,

Debian users suffer from a [poor LXQt support](https://packages.debian.org/sid/pcmanfm-qt). 

[Arch's](https://www.archlinux.org/packages/community/x86_64/pcmanfm-qt/) and OpenSUSE [Tumbleweed's](https://software.opensuse.org/download.html?project=X11%3ALXQt&package=pcmanfm-qt) LXQt supports are the best because LXQt members are active there.

**Comment by wxl** [comment]
> Debian users suffer from a poor LXQt support.

I'm hoping we can fix that soon. Meanwhile, at least downstream Ubuntu should be resolved soonish.

**Comment by tsujan** [comment]
> Meanwhile, at least downstream Ubuntu should be resolved soonish.

If Ubuntu 20.04 can have 0.15, it'll be great. If not, it's understandable because Ubuntu isn't a rolling distro.

As far as releases are signed by me, I guarantee that they won't be delayed anymore, in the sense that a lot of work won't be piled up only for git users -- to put it simply, this will really mean a support problem: "Your LXQt installation is outdated."

**Comment by franzos** [comment]
How can we resolve this situation? It's perfectly OK for a maintainer (or author) to be busy, but it hurts the projects reputation and progress for everything to go offline for a month. We (the PantherX Team) would be happy to provide funds or resources to bring everything back online, and remain online.

In any case, there should be minimum two trusted collaborators with access to:
1. Server
2. Domain (+DNS)

**Comment by tsujan** [comment]
> it hurts the projects reputation and progress for everything to go offline for a month.

Nothing has gone offline -- except for LXQt forum, of course. The releases will be more orderly than before.

I, for one, prefer to depend on nothing except for GitHub (and if GitHub does weird things because of Coronavirus -- as has become a routine nowadays -- GitLab). But I'll agree with whatever decision the LXQt team might make.

**Comment by franzos** [comment]
> Nothing has gone offline -- except for LXQt forum

Everything **except** GitHub is down and it appears to cause a lot of confusion. The majority of users might not even bother to look further, and just assume to project to be dead, or not worth a second look. At the very least, the _domains_ should redirect to GitHub.

> I, for one, prefer to depend on nothing except for GitHub

While I absolutely despise GitHub, having something up - reliably - is a plus. GitLab would ultimately be the same, unless the team is running a self hosted instance.

>  But I'll agree with whatever decision the LXQt team might make.

Who exactly is that "team"?

**Comment by jubalh** [comment]
> I, for one, prefer to depend on nothing except for GitHub

I agree on this.

> Everything except GitHub is down and it appears to cause a lot of confusion. The majority of users might not even bother to look further, and just assume to project to be dead, or not worth a second look. At the very least, the domains should redirect to GitHub.

We are aware. It doesn't change the fact that now we don't have access to the lxqt.org domain. And the person who has access vanished. If we can reach him again we will ask to forward the side, we don't need other people telling this to use we are aware of this ourselves.
In any case we will change all the URLs to the new GitHub URL and use this as our main URL. So packagers for various distributions should change their URL in case they have it in their packages.

Users assuming the project to be dead will eventually learn that it is not. Latest once they update their distribution and get a new LXQt ;)

We have no other possiblitities to do anything about it, if we would, we would have done it.

> Who exactly is that "team"?

The people in the lxqt GitHub organization.

**Comment by tsujan** [comment]
> Everything except GitHub is down

And GitHub is the only place we do team development and make releases. So, the rest isn't important *in practice*.

> While I absolutely despise GitHub...

Here, we rely on what's reliable. GitHub has been so, even after Microsoft owned it.

> GitLab would ultimately be the same

I agree but, for now, GitHub is the most well-known and the most active.

> Who exactly is that "team"?

→ "Member" tag on their comments.

**Comment by tsujan** [comment]
> If we can reach him again we will ask to forward the side...

Before I kill him ;)

**Comment by franzos** [comment]
@jubalh @tsujan thanks for clarifying!

**Comment by Borwe** [comment]
Still [lxqt.org](lxqt.org) dead, man, I was trying to checkout lxqt first time, trying to move from KDE to something else Qt based.

**Comment by tsujan** [comment]
@Borwe Read above! It'll be dead indefinitely and LXQt doesn't depend on it anymore.

**Comment by jubalh** [comment]
@Borwe still people who can't read. Man..

**Comment by Borwe** [comment]
Okay.

**Comment by wxl** [comment]
I'm pleased to see that Wikipedia is updated with the current URL and DuckDuckGo seems to return it as a top result. Google gave me Wikipedia as a first result, so that works out ok.

There are lots of references to it around the Internet, including within packages and wikis that distros maintain. I would urge you all to poke around your distro of choice and see if you can't work at resolving that. 

Then if we do get a hold of lxqt.org again, we could always redirect it at the GitHub page and no matter what, it will Just Work™.

**Comment by tsujan** [comment]
> Then if we do get a hold of lxqt.org again,

Then 3 LXQt members should have admin privileges, to prevent this from happening again. But I think we should keep things manageable, as they're now -- I won't be one of the admins.

**Comment by EinarMostad** [comment]
Hi! I am translating LXQt to Norwegian Bokmål, but now that the site is down, where do I contribute my translations? What about new strings that need translation? Do you have the translation saved somewhere else as well so you can build new versions of LXQt with translation or is all our work lost (like it was when Lubuntu.me effed up their translation website)?

**Comment by tsujan** [comment]
@EinarMostad 
Translators translate LXQt apps and their works are merged. Translation files are updated once in a while. Not only no work is lost but also more translations have been added.

For translating or updating translations, use Qt Linguist and make PRs.
